package.json

```diff
@@ -38,8 +38,8 @@
     "@babel/preset-typescript": "7.10.4",
     "@babel/register": "7.10.5",
     "@babel/runtime": "7.10.5",
-    "@changesets/changelog-github": "^0.1.1",
-    "@changesets/cli": "^2.10.1",
+    "@changesets/changelog-github": "0.1.1",
+    "@changesets/cli": "2.9.2",
```

Comment:
We are downgrading here , the original is `^2.10.1`

Reply:
fixed